packages/@o3r/rules-engine/src/services/runner/rules-engine.runner.service.spec.ts

```diff
@@ -94,22 +153,23 @@ describe('Rules engine service', () => {
   });
 
   it('Should support Block with no condition', async () => {
-    store.dispatch(setRulesetsEntities({entities: jsonOneRulesetOneRuleNoCond.ruleSets}));
+    store.dispatch(setRulesetsEntities({ entities: jsonOneRulesetOneRuleNoCond.ruleSets }));
     const actions = await firstValueFrom(service.events$);
     expect(actions.length).toBe(1);
     expect(actions[0].actionType).toBe('UPDATE_LOCALISATION');
   });
 
-  it('should handle linked components and validity range properly', (done) => {
-    service.events$.pipe(take(1)).subscribe((actions) => {
-      expect(actions.length).toBe(1);
-      expect(actions[0].actionType).toBe('UPDATE_LOCALISATION');
-      done();
+  it('should handle linked components and validity range properly', () => {
+    return new Promise((done) => {
+      service.events$.pipe(take(1)).subscribe((actions) => {
+        expect(actions.length).toBe(1);
+        expect(actions[0].actionType).toBe('UPDATE_LOCALISATION');
+        done();
+      });
+      store.dispatch(setRulesetsEntities({ entities: jsonOneRulesetValidOneRuleNoCond.ruleSets }));
     });
```

Comment:
```suggestion
  it('should handle linked components and validity range properly', async () => {
    const eventsPromise = firstValueFrom(service.events$);
    store.dispatch(setRulesetsEntities({ entities: jsonOneRulesetValidOneRuleNoCond.ruleSets }));
    const actions = await eventsPromise ;
    expect(actions.length).toBe(1);
    expect(actions[0].actionType).toBe('UPDATE_LOCALISATION');
```

Reply:
We deactivated the linter for this test to not refactor the test in this PR